climakitae/warming_levels.py

```diff
@@ -85,18 +85,18 @@ def set_subarea(boundary_dataset):
     elif location.area_subset == "CA watersheds":
         shape = set_subarea(location._geographies._ca_watersheds)
     ds_region = [shape]
-    
+
     #==================================
-    
-    # Un-list attributes so rioxarray can find them when it looks for a crs 
+
+    # Un-list attributes so rioxarray can find them when it looks for a crs
     proj = "Lambert_Conformal"
     for attr_np in ["earth_radius","latitude_of_projection_origin","longitude_of_central_meridian"]:
         postage_data[proj].attrs[attr_np] = postage_data[proj].attrs[attr_np].item()
-    
-    # Add grid-mapping attr (missing from Rel Humidity) 
-    if "grid_mapping" not in postage_data.attrs: 
-        postage_data.attrs["grid_mapping"] = proj 
-    
+
+    # Add grid-mapping attr (missing from Rel Humidity)
+    if "grid_mapping" not in postage_data.attrs:
```

Comment:
Is this missing because RH is a derived variable for hourly? If so, it would be better to fix this upstream so any downstream code doesn't need this check.

Reply:
Got it, thanks for the explanation. I see that grid mapping is assigned for [other vars](https://github.com/cal-adapt/climakitae/blob/main/climakitae/derive_variables.py#L48), so it should just be a matter of adding RH to that list, but I'll make a separate issue for that.